Automattic/newspack-ads
# Add semantic release

### Changes proposed in this Pull Request:



- Configures semantic-release for automated releases
- Configures JS & CSS linting
- Configures CircleCI to run the linting and releasing
- Removes `xwp/wp-dev-lib` in favour of `lint-staged` for pre-commit PHP linting (it's more efficient)
- Sets the PHP version for PHPCS linter to `7.2`

### How to test the changes in this Pull Request:

1. `npm run lint` should exit without error
2. `npm run format:js` & `npm run format:scss` & `npm run format:php` should not change any files
